packages/app-project/stores/User/UserPersonalization/UserPersonalization.js

```diff
@@ -46,10 +46,8 @@ const UserPersonalization = types
   }))
   .actions(self => {
     function _onUserChange() {
-      const { project, user } = getRoot(self)
-      if (user.id) {
-        self.notifications.fetchAndSubscribe()
-      } else if (user.loadingState === asyncStates.success) {
+      const { user } = getRoot(self)
+      if (user.loadingState === asyncStates.success) {
         self.projectPreferences.setLoadingState(asyncStates.success)
```

Comment:
This will mark project prefs as loaded while we’re still waiting for them to load, for a logged-in user. 

Personally I find the logic here confusing, but this is a workaround to set prefs as loaded for anonymous users.

Reply:
Also, this change should have caused tests to fail. 🤔

Reply:
My understanding on why we want user project preferences set to success for an anonymous user - I think it's because we use `appLoadingState` in components including [the ClassifierWrapper](https://github.com/zooniverse/front-end-monorepo/blob/master/packages/app-project/src/screens/ClassifyPage/components/ClassifierWrapper/ClassifierWrapper.js#L93) and [the ClassifyPage](https://github.com/zooniverse/front-end-monorepo/blob/master/packages/app-project/src/screens/ClassifyPage/ClassifyPage.js#L25), which needs the user project preferences loading state to `= 'success'` [per this code in the Store](https://github.com/zooniverse/front-end-monorepo/blob/master/packages/app-project/stores/Store.js#L23), and for a user we want the classifier to wait for user project preferences (UPPs) and for an anonymous user we want to load the classifier but there aren't UPPs so in the UserPersonalization store here we're forcing the UPPs loadingState to success so the classifier can load for an anonymous user. Not ideal, but I think makes sense.

No point/question to the above, just documenting my understanding.

Reply:
That’s my understanding too. There are some mocks in the tests that don’t work unless you explicitly set the loading state. I’ll see if I can find an example.